source/synthDrivers/_espeak.py

```diff
@@ -27,10 +27,10 @@
 _numBytesPushed = 0
 
 #Parameter bounds

```

Comment:
To make sure we keep in sync with eSpeak, it would be ideal to refer to some documentation.
There is some documentation [here](https://github.com/espeak-ng/espeak-ng/blob/467f3f565f9f8d6d1de62b854e3c2b21a52fd8a2/emscripten/README.md), but it's not ideal as its for the js library.
In absence of good documentation for these parameters, we could reference the code or https://github.com/espeak-ng/espeak-ng/issues/131.
What do you think is better?

```suggestion
# Parameter bounds
# For rate, refer to eSpeak variables espeakRATE_MINIMUM, espeakRATE_MAXIMUM.
# https://github.com/espeak-ng/espeak-ng/blob/61efed30fa1267336650d7217466156aa2177e80/src/include/espeak-ng/speak_lib.h#L82-L85
```

Reply:
I am a bit concerned by the suggestion of espeakng.js's documentation or a direct link to the code (src/include/espeak-ng/speak_lib.h) since these two sources indicate a max rate of 450 for the rate, whereas we stick to 449.
Thus I have just linked https://github.com/espeak-ng/espeak-ng/issues/131. If this issue is progressed in the future, we will have further indications to update NVDA from there.

Modification done in commit 32e9aaeb2fe306735fec37b7175bf476fbd2fde7.

Reply:
Thanks, makes sense